editorial nits (#49)
NLnetLabs/draft-toorop-dnsop-dns-catalog-zones: draft-ietf-dnsop-dns-catalog-zones.md

Catalog consumers SHOULD ignore any RR in the catalog zone which is meaningless or useless to the implementation.

> Authoritative servers may be pre configured with multiple catalog zones, each associated with a different set of configurations.

Reviewer: This seems wrong

> Authoritative servers may be pre configured with multiple catalog zones, each associated with a different set of configurations.

Reviewer 2: ```suggestion
Authoritative servers may be preconfigured with multiple catalog zones, each associated with a different set of configurations.
```

The PTR RRset MUST consist of a single PTR record.
More than one record in the RRset denotes a broken catalog zone which MUST NOT be processed (see (#generalrequirements)).

When a consumer of a catalog zone `$OLDCATZ` receives an update which adds or changes a `coo` property for a member zone in `$OLDCATZ`, it does *not* migrate the member zone immediately.
The migration has to wait for an update of `$NEWCATZ`. in which the member zone is present. The consumer MUST verify, before the actual migration, that `coo` property pointing to `$NEWCATZ` is still present in `$OLDCATZ`.

> Unless the member node label (i.e. `<unique-N>`) for the member is the same in `$NEWCATZ`, all associated states for a just migrated zone MUST be reset (see (#zonereset)).

Reviewer 2: ```suggestion
Unless the member node label (i.e. `<unique-N>`) for the member is the same in `$NEWCATZ`, all associated state for a just migrated zone MUST be reset (see (#zonereset)).
```

As with regular zones, primary and secondary nameservers for a catalog zone may
be operated by different administrators.  The secondary nameservers may be
configured as a catalog consumer to synchronize catalog zones from the primary, but the primary's
administrators may not have any administrative access to the secondaries.

Administrative control over what zones are served from the configured name servers shifts completely from the server operator (consumer) to the "owner" (producer) of the catalog zone content.

> With migration of member zones between catalogs using the `coo` property, it is possible for the owner of the target catalog (i.e. `$NEWCATZ`) to take over all associated states with the zone from the original owner (i.e. `$OLDCATZ`) by maintaining the same member node label (i.e. `<unique-N>`).

Reviewer 2: ```suggestion
With migration of member zones between catalogs using the `coo` property, it is possible for the owner of the target catalog (i.e. `$NEWCATZ`) to take over all associated state with the zone from the original owner (i.e. `$OLDCATZ`) by maintaining the same member node label (i.e. `<unique-N>`).
```
